### DISCUSSION TITLE
mentions plugin

### DISCUSSION THREAD

**Discussion by zgrybus** [Milkdown/milkdown#1280] [General]
Hello @Saul-Mirone 
I've created https://github.com/zgrybus/milkdown-mentions-plugin. May you include it in the https://github.com/Milkdown/awesome? :)

**Comment by Saul-Mirone** [comment]
Nice work! Done by https://github.com/Milkdown/awesome/commit/9f9cdf533d0138663a44db2bddc5a462dd86677d
